api/envoy/config/filter/udp/dns_filter/v2alpha/dns_filter.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package envoy.config.filter.udp.dns_filter.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+import "envoy/data/dns/v2alpha/dns_table.proto";
+
+import "udpa/annotations/status.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.udp.dns_filter.v2alpha";
+option java_outer_classname = "DnsFilterProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filter.udp.dns_filter.v3alpha";
+option (udpa.annotations.file_status).work_in_progress = true;
+
+// [#protodoc-title: DNS Filter]
+// DNS Filter :ref:`configuration overview <config_udp_listener_filters_dns_filter>`.
+// [#extension: envoy.filters.udp_listener.dns_filter]
+
+// Configuration for the DNS filter.
+message DnsFilterConfig {
+  // This message contains the configuration for the Dns Filter operating
+  // in a server context. This message will contain the virtual hosts and
+  // associated addresses with which Envoy will respond to queries
+  message ServerContextConfig {
+    oneof config_source {
+      // Load the configuration specified from the control plane
+      data.dns.v2alpha.DnsTable inline_dns_table = 1;
+
+      // Seed the filter configuration from an external path
```

Comment:
What is the data format of the external path? Binary proto? JSON/YAML/? Supports both? Can you clarify?

Reply:
It should be a yaml file.  I can update the comment.